From 2a7a933d971c30d3d8332fc12ecb40a9b8327d53 Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Thu, 17 Jan 2013 14:20:08 +0800
Subject: [PATCH 08/95] ENGR00214328 disabe support setting video sizes.

because it will set different resolution for preview and recorder.
but driver does not support this function now.

Signed-off-by: Xiaowen Liu <b37945@freescale.com>

Conflicts:
	services/camera/libcameraservice/api1/client2/Parameters.cpp
---
 services/camera/libcameraservice/api1/client2/Parameters.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/services/camera/libcameraservice/api1/client2/Parameters.cpp b/services/camera/libcameraservice/api1/client2/Parameters.cpp
index 1e10320..6cf737b 100644
--- a/services/camera/libcameraservice/api1/client2/Parameters.cpp
+++ b/services/camera/libcameraservice/api1/client2/Parameters.cpp
@@ -141,8 +141,6 @@ status_t Parameters::initialize(const CameraMetadata *info, int deviceVersion) {
                     availableVideoSizes[i].height);
         }
         ALOGV("Supported video sizes are: %s", supportedVideoSizes.string());
-        params.set(CameraParameters::KEY_SUPPORTED_VIDEO_SIZES,
-                supportedVideoSizes);
     }
 
     camera_metadata_ro_entry_t availableFpsRanges =
-- 
1.8.0

